docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
-Enable Nvidia GPUs
+.. _nvidia-gpu:
+
+Enable NVIDIA GPUs
 ==================
 
-To be able to run workloads on GPU with DSS, you need to have:
+The following section describes how to configure the DSS to utilize the host's
+NVIDIA GPU. This is achieved by configuring the underlying MicroK8s, that the
```

Comment:
As a rule of thumb, better to have whole sentences in the same line (when possible) - that way is easier for reviews/updates.

Reply:
@afgambin I think you can select multiple lines for which to suggest changes (click on first line and then Shift and click on the next line)

Would this help?